APMrover2/APMrover2.cpp

```diff
@@ -95,6 +95,7 @@ const AP_Scheduler::Task Rover::scheduler_tasks[] = {
 #if ADVANCED_FAILSAFE == ENABLED
     SCHED_TASK(afs_fs_check,           10,    200),
 #endif
+    SCHED_TASK(windvane_update,        10,    200),
```

Comment:
Use `SCHED_TASK_CLASS` rather then provide a one line helper function.

Reply:
yup, this crossed my mind as well.  txs for pointing this out, replaced windvane_update with direct call from scheduler.